internal/server/singleprocess/service_ui_deploy.go

```diff
@@ -106,7 +119,16 @@ func (s *service) UI_ListDeployments(
 					(&ptypes.Deployment{Deployment: deploy}).URLFragment(),
 					strings.TrimPrefix(hostname.Fqdn, hostname.Hostname),
 				)
+
+				deployURLName = hostname.Hostname
+				deployURLHost = strings.TrimPrefix(hostname.Fqdn, hostname.Hostname)
 			}
+		} else {
+			bundle.DeployUrl = fmt.Sprintf(
+				"%s--%s%s",
+				deployURLName,
+				(&ptypes.Deployment{Deployment: deploy}).URLFragment(), // Deployment Sequence Number
+				deployURLHost)
```

Comment:
I might be reading this wrong, but I think if `deploy.HasEntrypointConfig == false` then we’d end up executing this `else` clause and trying to set `bundle.DeployUrl` even though there’s no `deployURLHost` to work with.

Reply:
Done!